website/docs/guides/docs/docs-create-doc.mdx

```diff
@@ -174,6 +176,29 @@ It is possible to use:
 
 :::
 
+#### Relationship between `id` and `slug` and document path
+
+- The `id` front matter defines the unique identifier for the document within Docusaurus. By default, `id` is derived from the file path and name.
+- The `slug` front matter controls the published URL path of the document. Changing `id` alone does **not** affect the URL; only `slug` changes the URL.
```

Comment:
confusing: changing id alone can definitively affect the url. It only does not affect it when you provide an explicit slug.